### PR TITLE
Update LICENSE to Brightcove, Inc and Apache 2.0

### PR DESCRIPTION
**Pull request by gesinger**

### CODE REVIEW

**Comment by forbesjo** [comment]
LGTM 💯

File: bower.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "videojs-contrib-eme",
   "version": "0.0.0",
```
Comment (imbcmdth) [inline bower.json]: Should this get a version, now?

Comment (gkatsev) [inline bower.json]: `bower.json` shouldn't have a version field at all.

File: package.json [hunk 2]
```diff
@@ -37,8 +37,9 @@
     "videojs",
     "videojs-plugin"
   ],
-  "author": "Garrett Singer <gesinger@gmail.com>",
-  "license": "MIT",
+  "copyright": "Copyright Brightcove, Inc. <https://www.brightcove.com/>",
+  "license": "Apache-2.0",
+  "author": "Brightcove, Inc.",
```
Comment (gkatsev) [inline package.json]: you can keep your name here if you want. Or just remove it altogether.

**Pull request merged**